test/Interop/C/swiftify-import/counted-by.swift

```diff
@@ -57,19 +66,28 @@ import CountedByClang
 // CHECK-NEXT: @_alwaysEmitIntoClient @_disfavoredOverload public func simpleFlipped(_  p: UnsafeMutableBufferPointer<Int{{.*}}>)
 
 // CHECK-NEXT: /// This is an auto-generated wrapper for safer interop
-// CHECK-NEXT: @_alwaysEmitIntoClient @_disfavoredOverload public func sizeofParam(_  p: UnsafeMutableBufferPointer<Int{{.*}}>)
```

Comment:
`sizeof` results in an unsigned integer literal expression, so `sizeofParam` and `sizeofType` were always broken, but we didn't call the functions so they were never type checked.